client/syntaxes/yml.tmLanguage.json

```
@@ -24,7 +24,7 @@
         "keywords": {
             "patterns": [{
                     "name": "keyword.control.yml",
-                    "match": "\\b(if|while|for|return|foreach|switch|switchExpr|switchExprExclusive|belongsTo|default|case|interface|class|implementation|private|extends|override|complete|field|method|function|static|classProperties|ifExpr|then|else)\\b",
+                    "match": "\\b(if|while|for|return|foreach|switch|switchExpr|args|switchExprExclusive|belongsTo|default|case|interface|class|implementation|private|extends|override|complete|field|method|function|Function|static|local|classProperties|ifExpr|then|else|break)\\b",
```

The order is getting more and more confusing. :thinking:

```
$ tr '|' '\n' <<< 'if|while|for|return|foreach|switch|switchExpr|args|switchExprExclusive|belongsTo|delt|case|interface|class|implementation|private|extends|override|complete|field|method|function|Function|static|local|classProperties|ifExpr|then|else|break' | sort --ignore-case | tr '\n' '|' | sed 's,|$,\n,'
args|belongsTo|break|case|class|classProperties|complete|default|else|extends|field|for|foreach|function|Function|if|ifExpr|implementation|interface|local|method|override|private|return|static|switch|switchExpr|switchExprExclusive|then|while
```

```suggestion
                    "match": "\\b(args|belongsTo|break|case|class|classProperties|complete|default|else|extends|field|for|foreach|function|Function|if|ifExpr|implementation|interface|local|method|override|private|return|static|switch|switchExpr|switchExprExclusive|then|while)\\b",
```

My work on Prism.js made me realize that once lists grow to a certain size, the alphabetic order is generally more helpful than most attempts at semantical organization.